app/services/analytics.rb

```diff
@@ -51,7 +49,7 @@ def uuid
   INVALID_AUTHENTICITY_TOKEN = 'Invalid Authenticity Token'.freeze
   INVALID_SERVICE_PROVIDER = 'Invalid Service Provider'.freeze
   OTP_DELIVERY_SELECTION = 'OTP: Delivery Selection'.freeze
-  OTP_RESULT = 'OTP: Result'.freeze
+  MFA = :mfa
```

Comment:
+1 to clearer names, at least in the value (ex `MFA = 'Multi-Factor Auth'.freeze` or something)

Also I though the point of pulling everything into this file was so we could be more consistent with names? I'd expect 100% full names, and also 100% strings? If we want to be 100% symbols that's fine but I definitely think we should not be mixing and matching